scripts/upload-certs.sh

```diff
@@ -92,26 +97,24 @@ main() {
         exit 1
     fi
 
-    local certificate
-    certificate=$(sed 's/$/\\n/' "${keypath}/f0000013.cert" | tr -d \\n)
-    local signup_pubkey
-    signup_pubkey=$(sed 's/$/\\n/' "${keypath}/sss_70000002_0002_0040.bin" | tr -d \\n)
-    local attestation_pubkey
-    attestation_pubkey=$(sed 's/$/\\n/' "${keypath}/sss_70000001_0002_0040.bin" | tr -d \\n)
-
-    # Get Cloudflared token
     echo "Getting Cloudflared access token..."
     local cf_token
     cf_token="$(get_cloudflared_token "${domain}")"
 
-    # Post certificate
-    curl --fail --location \
-        -H "Authorization: Bearer ${bearer}" \
-        -H "cf-access-token: ${cf_token}" \
-        -X POST "https://${domain}/api/v1/certificate" \
-        -d '{ "orbId": "'"${orb_id}"'", "certificate": "'"${certificate}"'" }'
+    # Post attestation certificate
+    if [[ ! "${short}" ]]; then
```

Comment:
Another usage feedback that I had is that a lot of times dir name is the orb-id, so we don't need to provide that explicitely. We can use that in next script that combines both copy/upload.